long_term/extract.py

```diff
@@ -0,0 +1,67 @@
+"""Extracts all of the json files from the S3 bucket more than 24h old."""
+
+import os
```

Comment:
These should likely be separated into standard library, third party, and local libraries.